web/pgadmin/browser/server_groups/servers/databases/schemas/statistics/tests/test_statistics_get.py

```
@@ -0,0 +1,157 @@
+                self.assertIn('stat_types', response_data)
+        else:
+            if 'statistics_id' in self.data:
+                self.statistics_id = self.data["statistics_id"]
+            get_call = (
+                statistics_utils.api_get(self, '')
+                if self.is_list else statistics_utils.api_get(self)
+            )
+            if self.mocking_required:
+                with patch(
+                    self.mock_data["function_name"],
+                    side_effect=[eval(self.mock_data["return_value"])]
```

Ruff (0.16.1)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)